doc/classes/Particles.xml

```diff
@@ -55,8 +55,10 @@
 			Time ratio between each emission. If [code]0[/code] particles are emitted continuously. If [code]1[/code] all particles are emitted simultaneously. Default value: [code]0[/code].
 		</member>
 		<member name="fixed_fps" type="int" setter="set_fixed_fps" getter="get_fixed_fps">
+			The particle system's frame rate is fixed to a value. For instance, changing the value to 2 will make the particles render at 2 frames per second. Note this does not slow down the particle system itself.
```

Comment:
Nitpick suggestion: `Note this does not slow down the particle system itself.` -> `Note this does not slow down the simulation of the particle system itself.`

Reply:
Good call. This one was just copied from the existing text.